Dockerfile

```diff
@@ -54,11 +54,39 @@ RUN mkdir -p /artifacts && \
     /root/.cargo/bin/cargo build --profile=release && \
     cp target/release/libcorerust.so /artifacts/libcorerust.so
 
-FROM radixdlt/babylon-node-build-layers:${IMAGE_VERSION}-app AS main
+# =================================================================================================
+# LAYER: library-container
+# A layer containing just the built library at the root: /libcorerust.so
+# =================================================================================================
+FROM scratch AS library-container
+COPY --from=library-build-stage /artifacts/libcorerust.so /
+
+# =================================================================================================
+# LAYER: java-container
+# Exports only the java application artifacts from the java-build-stage
+# =================================================================================================
+FROM scratch AS java-container
+COPY --from=java-build-stage /radixdlt/core/build/distributions /
+
+# =================================================================================================
+# LAYER: library-builder-local
+# This layer is just used for local building in development via `local-cached-rust-build.yaml`.
```

Comment:
This comment hasn't been true for a few years - in local development, we run a native build.

I think this layer is just used for building the Rust image for building a debian package.